packages/flutter/lib/src/cupertino/tab_scaffold.dart

```diff
@@ -120,6 +201,12 @@ class CupertinoTabScaffold extends StatefulWidget {
   /// Must not be null.
   final CupertinoTabBar tabBar;
 
+  /// Controls the current selected tab item of the [tabBar], as well as the
```

Comment:
'Item' sounds like this controller controls the BottomNavigationBarItems of the tab bar. Say tab index or something.